src/pages/[special].astro

```
@@ -0,0 +1,47 @@
+---
+import DefaultLayout from '@layouts/DefaultLayout.astro'
+import PageHeader from '@components/PageHeader.astro'
+import SocialShares from '@components/SocialShares.astro'
+import { getCollection, render } from 'astro:content'
+import type { CollectionEntry } from 'astro:content'
+
+export async function getStaticPaths() {
+  const special = await getCollection('special')
+
+  return special.map((s, index) => {
```

ESLint

[error] 'index' is defined but never used.

(no-unused-vars)